### ISSUE TITLE
Wrong paths in PyIlmBase pkg-config for 2.5.5

### ISSUE THREAD

**Issue by waebbl** [AcademySoftwareFoundation/openexr#922]
This seems to be partly a regression of issue #678

The paths set in `PyIlmBase/config/CmakeLists.txt` in function `pyilmbase_pkg_config_help` are not creating a correct PyIlmBase.pc file. For Gentoo, I'm using the patch below to fix it:

```
diff --git a/config/CMakeLists.txt b/config/CMakeLists.txt
index 1872c89..44a9f2f 100644
--- a/config/CMakeLists.txt
+++ b/config/CMakeLists.txt
@@ -15,9 +15,9 @@ if(PYILMBASE_INSTALL_PKG_CONFIG)
   # use a helper function to avoid variable pollution, but pretty simple
   function(pyilmbase_pkg_config_help pcinfile)
     set(prefix ${CMAKE_INSTALL_PREFIX})
-    set(exec_prefix ${CMAKE_INSTALL_BINDIR})
-    set(libdir ${CMAKE_INSTALL_LIBDIR})
-    set(includedir ${CMAKE_INSTALL_INCLUDEDIR})
+    set(exec_prefix "\${prefix}")
+    set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
+    set(includedir "\${exec_prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
     set(LIB_SUFFIX_DASH ${OPENEXR_LIB_SUFFIX})
     string(REPLACE ".in" "" pcout ${pcinfile})
     configure_file(${pcinfile} ${CMAKE_CURRENT_BINARY_DIR}/${pcout} @ONLY)
```

**Comment by cary-ilm** [comment]
This should be resolved in the v3.0.0 release, but let us know if a patch for 2.5 is essential.

**Comment by waebbl** [comment]
It's working for v3.0.0. For v2.5.5, i've added the patch to the build script, which is ok.
However, depending on how long you plan to support the v2.5 release, it might be worth thinking about backporting the patch.
